lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -6,15 +6,16 @@
             'gettext',
             'js/views/fields',
             'teams/js/models/team',
+            'common/js/components/utils/view_utils',
             'text!teams/templates/edit-team.underscore'],
-        function (Backbone, _, gettext, FieldViews, TeamModel, editTeamTemplate) {
+        function (Backbone, _, gettext, FieldViews, TeamModel, ViewUtils, editTeamTemplate) {
             return Backbone.View.extend({
 
                 maxTeamNameLength: 255,
                 maxTeamDescriptionLength: 300,
 
                 events: {
-                    'click .action-primary': 'createOrUpdateTeam',
+                    'click .action-primary': function(event) {this.disableRun(event, this.createOrUpdateTeam);},
```

Comment:
@efischer19 Sorry to do this at this point, but the solution you have still doesn't end up the way I was suggesting in this thread here. I think it is important that this can be added just by wrapping the event, and that it doesn't require an extra function in order to make things work. The trick as I mentioned is to move the anonymous function into the helper method.

I've pushed up a commit that converts what you have to the syntax that I was proposing (my commit is currently 537280f373218d6e466cc3798f489dc0ff483ca2). It is now:

    'click .action-primary': ViewUtils.withDisabledElement('createOrUpdateTeam')

Let's discuss first thing in the morning and make sure that I'm not missing something.